AUTHORS

```diff
@@ -747,6 +747,7 @@ answer newbie questions, and generally made Django that much better:
     Sam Newman <http://www.magpiebrain.com/>
     Sander Dijkhuis <sander.dijkhuis@gmail.com>
     Sanket Saurav <sanketsaurav@gmail.com>
+    Sanyam Khurana <sanyam.khurana01@gmail.com>
```

Comment:
As per https://github.com/django/django/pull/10652/files#r234646818:

> Ooops. As Tim just pointed out on #10652, this isn't in the right place. (We only need it once 🙂)

Reply:
The one in this PR is sorted alphabetically. I will remove it from other PRs. Sounds good?

Reply:
Yeah, fine. 
Also can you [adjust the commit message](https://docs.djangoproject.com/en/dev/internals/contributing/committing-code/#committing-guidelines) here too?

Reply:
@carltongibson I've reframed the commit message.